web/src/main/java/com/bazaarvoice/emodb/web/scanner/config/ScheduledScanConfiguration.java

```diff
@@ -3,6 +3,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Optional;
 import com.google.common.collect.ImmutableList;
+import io.dropwizard.util.Duration;
```

Comment:
I'd recommend simplifying by just using `org.joda.time.Duration`; that's practically what's being done anyway by the only method which uses it, `getMaxRangeScanTime()`.  The only change is that the config file would use `PT10M` as the value instead of the more DropWizardy `10m`

Reply:
I am totally fine with doing that, the only issue is I don't think that org.joda.time.Duration has Jackson bindings, so we will have to make the config value initially deserialized as a String

Reply:
It does, see `com.fasterxml.jackson.datatype.joda.JodaModule`.